kola/tests/ignition/resource.go

```diff
@@ -128,6 +128,22 @@ func init() {
 		  }
 	      }`),
 	})
+	register.Register(&register.Test{
```

Comment:
Would it make sense to combine this with `coreos.ignition.v2_1.resource.s3`?  It doesn't seem as though a separate test is necessary.

Reply:
SGTM